service/src/main/java/com/dima/http/rest/UserRestController.java

```diff
@@ -0,0 +1,64 @@
+package com.dima.http.rest;
+
+import com.dima.dto.PageResponse;
+import com.dima.dto.UserCreateEditDto;
+import com.dima.dto.UserReadDto;
+import com.dima.filter.UserFilter;
+import com.dima.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/users")
+public class UserRestController {
+
+    private final UserService userService;
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
consumer and produce are APPLICATION_JSON_VALUE by default - no need to set it up explicitly